### PR TITLE
Grafna deployment update.

### PR DESCRIPTION
**Pull request by mrsiano**
updates:
- automate the oauth configuration phase.
- deploy node exporter dashboard.
- use bash switches.
- update readme fonts and how to.

### CODE REVIEW

File: examples/grafana/README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-```
+## Available Dashboards
+- openshift cluster metrics
+- node exporter metrics
 
-
-## To deploy grafana
-Note: make sure to have openshift prometheus deployed.
+## To run grafana and deploy dashboards
+Note: make sure to have openshift prometheus deployed and or node exporter.
```
Comment (soltysh) [inline examples/grafana/README.md]: Suggestion: ... prometheus deployed (possibly, with node exporter).

File: examples/grafana/README.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 (https://github.com/openshift/origin/tree/master/examples/prometheus)
 
-``` ./setup-grafana.sh prometheus ```
+### Run the deployment script
+``` 
+./setup-grafana.sh -n prometheus -a -e
```
Comment (soltysh) [inline examples/grafana/README.md]: Explain what `-n prometheus`, `-a`, and `-e` stand for.

**Comment by mrsiano** [comment]
@soltysh PTAL

**Pull request merged**